tables/hbo.tbl

```diff
@@ -0,0 +1,275 @@
+#https://en.wikipedia.org/wiki/Hebrew_Braille#:~:text=The%20International%20Hebrew%20Braille%20Code
+#also see: 
+#https://huc.edu/library_blog/learning-about-hebrew-braille/#:~:text=The%20Code%20created%20by%20Brevis,books%20written%20in%20Hebrew%20Braille.
+
+#-maintainer: Paul Geoghegan <contact@envisionly.tech>
+#-maintainer: Matityhau Yeshurun <yeatersink@gmail.com>
+
+# always op code characters
+always ִי 35  # hiriq-yod
+always ֵי 3456  # tsere-yod
+always וֹ 246  # holam-vav
+
+# digit op code characters
+digit 0 245  # 0
+digit 1 1  # 1
+digit 2 12  # 2
+digit 3 14  # 3
+digit 4 145  # 4
+digit 5 15  # 5
+digit 6 124  # 6
+digit 7 1245  # 7
+digit 8 125  # 8
+digit 9 24  # 9
+
+# letter op code characters
+letter ֒ 46-234  # segol - isolated
+letter ֓ 46-146  # shalshelet
+letter ֔ 2  # zaqef qatan
+letter ֕ 46-1245  # zaqef gadol
+letter ֖ 46-2345  # tipeha
+letter ֗ 46-1235  # revia
+letter ֘ 46-145  # zarqa
+letter ֙ 46-34  # pashta
+letter ֚ 46-245  # yetiv
+letter ֛ 46-1236  # tevir
+letter ֜ 46-1  # geresh
+letter ֝ 46-16  # geresh muqdam
+letter ֞ 46-12  # gershayim
+letter ֟ 46-1345  # qarney para
+letter ֠ 46-2356  # telisha gedola
+letter ֡ 46-236  # pazer
+letter ֢ 46-23  # atnah hafukh
+letter ֣ 46-136  # munah
+letter ֤ 46-125  # mahapakh
+letter ֥ 46-134  # merkha
+letter ֦ 46-13  # merkha kefula
+letter ֧ 46-145  # darga
+letter ֨ 46-1234  # qadma
+letter ֩ 46-12345  # telisha qetana
+letter ֪ 46-13456  # yerah ben yomo
+letter ֫ 46-135  # ole
+letter ֬ 46-24  # iluy
+letter ֭ 46-24  # dehi
+letter ֮ 46-1356  # zinor
+letter ֯ 456-134  # masora circle
+letter ְ 3  # sheva
+letter ֱ 26  # hataf segol
+letter ֲ 25  # hataf patah
+letter ֳ 345  # hataf qamats
+letter ִ 24  # hiriq
+letter ֵ 34  # tsere
+letter ֶ 15  # segol
+letter ַ 14  # patah
+letter ָ 126  # qamats
+letter ֹ 456-246  # holam Malay
+letter ֺ 135  # holam haser
+letter ֻ 346  # qubuts
+letter ּ 5  # dagesh
+letter ּו 136  # shuruq
+letter ּוְ 136-3  # shuruq sheva
+letter ֽ 4  # meteg
+letter ֿ 45  # rafe
+letter ׁ 456-4  # shin dot
+letter ׂ 456-45  # sin dot
+letter ׄ 456-5  # upper dot
+letter ׅ 456-56  # lower dot
+letter ׆ 456-1345  # nun hafukha
+letter ׇ 126  # qamats qatan
+letter א 1  # alef
+letter ב 1236  # bet
+letter בְּ 12-3  # bet with dagesh sheva
+letter בּ 12  # bet with dagesh
+letter ג 1245  # gimel
+letter ד 145  # dalet
+letter דּ 5-145  # dalet with dagesh
+letter דְּ 5-145-3  # dalet with dagesh sheva
+letter ה 125  # hey
+letter הּ 125  # hey - isolated
+letter ו 2456  # vav
+letter וּ 136  # shuruq
+letter ז 1356  # zayin
+letter ח 1346  # het
+letter ט 2345  # tet
+letter י 245  # yod
+letter יּ 5-245  # yod with dagesh
+letter יְּ 5-245-3  # yod with dagesh
+letter ך 16  # final kaf
```

Comment:
This is going to cause issues with back translation I think. I'm pretty sure back translation uses the first rule it finds, so it will back translate every dots 16 with the final kaf.
May be it's a good starting point to mark all final letters with `noback`. This means the finals aren't available for back translation, but I think it goes much beyond the current scope to support back translations of finals just yet.

Reply:
There is probably a way to make it possible to recognize final kaf using other opcodes.